Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -2,11 +2,12 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include <QApplication>
 #include <QDir>
 #include <QFileDialog>
 #include <QIcon>
 #include <QMessageBox>
-
+#include <qpa/qplatformnativeinterface.h>
```

Comment:
Qt [warns](https://github.com/qt/qtbase/blob/6bceb4a8a9292ce9f062a38d6fe143460b54370e/src/gui/kernel/qplatformnativeinterface.h#L47) about using this directly in applications.